src/Handler/AddPrefixCommandHandler.php

```diff
@@ -22,6 +26,30 @@
 class AddPrefixCommandHandler
 {
     /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var Finder
+     */
+    private $finder;
+
+    /**
+     * @var Scoper
+     */
+    private $scoper;
+
+    public function __construct(Filesystem $filesystem, Finder $finder)
```

Comment:
`Filesystem` and `Finder` are utility classes that are not meant to be replaced. I think you shouldn't inject them, but simply construct them internally.